tests/conftest.py

```diff
@@ -112,7 +121,10 @@ def testbed_devices(ansible_adhoc, testbed):
 
     devices = {
         "localhost": Localhost(ansible_adhoc),
-        "dut": SonicHost(ansible_adhoc, testbed["dut"], gather_facts=True)}
+        "dut": SonicHost(ansible_adhoc, testbed["dut"], gather_facts=True)
+    }
+
+    devices["fanout"] = FanoutHost(ansible_adhoc, devices["localhost"], devices["dut"])
```

Comment:
Can we create a dedicate fixture for the FanoutHost and do not include it in the testbed_devices fixture? My idea of creating this testbed_devices fixture was not good. It could get bigger and bigger like this. Some test scripts using this fixture may only need access to duthost. Then initializing other objects is a waste and causes overhead for running these tests. Probably I'll deprecate the testbed_devices fixture in the future.

Reply:
Fixed